upnext/tracks.py

```diff
@@ -41,9 +41,12 @@ def add_to_playlist(track_info, party):
 
     # Adding in DB
     try:
-        track_artist = re.search('artist\': u\'(.+?)\'', track_info).group(1)
+        track_artist = re.search('artist\': u\"(.+?)\"', track_info).group(1)
     except AttributeError:
-        print "Track artist not found"
+        try:
+            track_artist = re.search('artist\': u\'(.+?)\'', track_info).group(1)
+        except AttributeError:
+            print "Track artist not found"
```

Comment:
Yeah, so... fun times.

Sometimes when we get stuff back from Spotify, it looks like: `artist: "The Mowgli's"` (because apostrophe) and other times it's: `artist: 'Arctic Monkeys'` (no apostrophe). I need two blocks in order to catch them both. This was just the laziest way of doing it.

Otherwise I could do something like, try the first thing, and then `if track_artist == None` do the second one. But I don't know how that works with the regex stuff I have going on.